### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by Jineshdarjee**
## Description
- Added unit test cases for Arka EPV07

### CODE REVIEW

File: admin_frontend/src/constants/defaultNetworks.json
Comment (vignesha22) [inline admin_frontend/src/constants/defaultNetworks.json:1]: pls remove this changes as it is not required and we have made some changes with respect to epv07

File: backend/config.json.default
Comment (vignesha22) [inline backend/config.json.default:1]: pls revert this change

File: backend/demo.env
Comment (vignesha22) [inline backend/demo.env:1]: Pls revert this change as these are important files

File: backend/src/paymaster/index.ts
Comment (vignesha22) [inline backend/src/paymaster/index.ts:1]: pls revert this file changes too

File: admin_frontend/src/constants/defaultNetworks.json
Comment (Jineshdarjee) [reply admin_frontend/src/constants/defaultNetworks.json:1]: Done

File: backend/config.json.default
Comment (Jineshdarjee) [reply backend/config.json.default:1]: Done

File: backend/demo.env
Comment (Jineshdarjee) [reply backend/demo.env:1]: Done

File: backend/src/paymaster/index.ts
Comment (Jineshdarjee) [reply backend/src/paymaster/index.ts:1]: Done

File: backend/src/routes/index.ts
Comment (vignesha22) [inline backend/src/routes/index.ts:1]: pls revert this change too

Comment (Jineshdarjee) [reply backend/src/routes/index.ts:1]: DOne

**Review by vignesha22 — APPROVED** [review]
